src/hyperfine/export/csv.rs

```diff
@@ -0,0 +1,26 @@
+use super::{ExportEntry, ResultExporter};
+
+use csv::WriterBuilder;
+use std::io::Result;
+
+pub struct CsvExporter {
+    out_file: String,
+}
+
+impl ResultExporter for CsvExporter {
+    fn write(&self, results: &Vec<ExportEntry>) -> Result<()> {
+        let mut writer = WriterBuilder::new().from_path(&self.out_file)?;
+        for res in results {
+            writer.serialize(res)?;
+        }
+        Ok(())
+    }
+}
+
+impl CsvExporter {
+    pub fn new(file_name: String) -> Self {
```

Comment:
Could the file-handling part be generalized and moved to `export/mod.rs`? Otherwsise, every exporter type would have to handle this on its own. Instead, each exporter could probably just return a `String` with the file contents.

Reply:
It could, I wanted to let particular serializer libraries handle output to files in their own way. I can look into refactoring this later to unify file handling however